package.json

```diff
@@ -100,7 +100,7 @@
     "webpack": "^5.75.0"
   },
   "engines": {
-    "node": "14.* || 16.* || >= 18"
+    "node": "16.* || >= 18"
```

Comment:
this will be removed when I do the v2 conversion. no need for `engines`

Reply:
I don't see the point of merging a change that _cannot_ be released

Reply:
We can't do an incremental migration to a v2 addon _and_ have the main branch always be in a releasable state....

Reply:
@NullVoxPopuli your current approach with splitting it up is fine.

@kategengler we do this all the time for "up next" kinds of branches, and it's literally no different from shipping an Ember major. Indeed: we'll do the exact same thing we would on Ember itself here: `master` becomes the branch for the next major, and if we ID things that need back-porting we'll create a `release-2-x` branch and do the usual back port dance.

Reply:
@chriskrycho I agree we do this is in some projects, but most addons release from `master` and release much frequently. think it is fine, so long as anybody that may release knows that `master` is not in a release-able state.

Reply:
There's also a difference here in that dropping Node 14 would be a totally fine breaking major change for this project if we weren't concerned with using it in the CLI blueprint. That `engines` part of package.json will look normal and fine to anybody not thinking about CLI's Node support policy and could easily be accidentally released. 

I'd also like to push back on the idea that we merge things that are in a broken state in `ember.js`, even in a major. The canary branch of `ember.js` _is_ released weekly as an alpha, available as a nightly. We may have breaking changes queued up, but that only affects the version number with which it will be released.

Reply:
I’d be fine with targeting a `next` branch to ameliorate that concern, FWIW, but we should differentiate “broken”, which this is not, from “requires communication to avoid breaking folks if we’re not on the same page about release plans”, which you’re absolutely correct that this is.

Reply:
If you released this change CLI would not be able to update the addon, and anybody on Node 14 that took the update would get an error. Seems pretty broken to me? As @NullVoxPopuli said, the intention is this part of the change is never released.

Reply:
@NullVoxPopuli I don't think we need a multi-branch workflow for most addons. I think this is a special case because of `engines` and the CLI node policy. Except for policy-based things such as that, the main branch being releasable is enforced by CI. Personally, I would have left the less restrictive `engines` and just changed the development version of Node to make the move to a v2 addon so there wouldn't be a change that "we need to remember to undo". This is fine to move forward, we just need to remember to undo the engines change.

Reply:
My plan was was to remove `engines` entirely, in the 4th PR in this series

1. (this one)
2. the pnpm one, because yarn and peers don't work (I have evidence if you need it, also yarn doesn't even support yarn... so...)
3. test-app split
4. v2 addon and `engines` removal

Reply:
@NullVoxPopuli FWIW although I merged this to keep a logjam, I think the better move here (and I should have thought of this!) was: bump the *Volta* version while leaving `engines` alone until that 4th PR. That's what we do in Glint for a variety of reasons. Let's keep that in mind for future reference.

Reply:
Will do, for future addons, I'll do this series of PRs:

0. bump volta
1. pnpm 
2. test-app split
3. v2 addon and engines removal